api/controllers/main.js

```diff
@@ -34,4 +34,21 @@ module.exports = {
 
     res.render('index.html', context);
   },
+
+  robots(req, res) {
+    const PROD_CONTENT = 'User-Agent: *\nDisallow: /preview\n';
+    const DENY_ALL_CONTENT = 'User-Agent: *\nDisallow: /\n';
+
+    res.set('Content-Type', 'text/plain');
+
+    // If this is the production instance and the request came to the production hostname
+    if (config.app.app_env === 'production' &&
+      config.app.hostname === `${req.protocol}://${req.hostname}`) {
```

Comment:
I think we need to do `app.enable('trust proxy')` so that express will use `X-Forwarded-Host` as the `req.hostname` when it is available. CloudFront implements this safely -- it will not allow clients to set `X-Forwarded-*` headers.